### ISSUE TITLE
No Build menu access

### ISSUE THREAD

**Issue by washaldecaldo** [Neurolinked/MlsetupBuilder#9]
I can't get the Build menu to open, just shows this message about CLI. I downloaded the zip and extracted like usual, but I also tried via Wkit and got the same problem.

![image](https://user-images.githubusercontent.com/113277514/204068935-4aea3c84-df41-4b6e-898c-f0d338e76ca3.png)

**Comment by Neurolinked** [comment]
Have you setup in the preferences the wolvenkit.CLI executable ?

If you use Wolvenkit, to not interfere with the software access to files the feature "Repository building" is hidden and not usable by default

Wolvenkit still download the whole application and you can start it as standalone software finding it's folder and starting MLsetupBuilder from there.
To uncook the needed material you have to configure Wolvenkit.CLI

**Comment by washaldecaldo** [comment]
Sorry for delayed response. Yeah I have set it up, and I've used mlsetupbuilder before, since 1.3 I think. this just started happening, and even when I uninstall and reinstall it, or even try a previous version, it still does it. Are there preference files saved somewhere that I need to delete? 🤔

It works fine when I launch via editing json in wkit so I guess I finally adopted the modern workflow, so I'll close for now. If you have any ideas about using it outside of wkit that would be great -- sometimes I like to keep the file open bc I'm iterating quickly.
